### ISSUE TITLE
[Bug]: Secrets not loaded when using Base directory during the build

### ISSUE THREAD

**Issue by Aviortheking** [coollabsio/coolify#764]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

When setting up a repository with a Base directory other than the default Secrets are not loaded during build time

### Steps To Reproduce

1. Create a new Application with this public repo `https://github.com/Aviortheking/test_coolify`
2. add a secret `TEST_ENV`
3. Build it first with the default Base Directory
3.5. You should see in the build logs
```
[14:44:49.614] #11 [7/7] RUN printenv	&& echo pouet
[14:44:49.618] #11 sha256:5ab3bb18c08f22d13e48607d098e90ea40630e4bc42195289fbe8755e12c1076
[14:44:49.890] #11 0.425 NODE_VERSION=19.2.0
[14:44:49.896] #11 0.425 TEST_ENV=pouet
[14:44:49.901] #11 0.425 YARN_VERSION=1.22.19
[14:44:49.913] #11 0.425 SHLVL=1
[14:44:49.917] #11 0.425 HOME=/home/node
[14:44:49.922] #11 0.425 PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
[14:44:49.934] #11 0.425 PWD=/home/node
[14:44:49.941] #11 0.425 pouet
```

4. Change the base directory to sub
5. See that the env is not loaded
```
[14:44:07.194] #11 [7/7] RUN printenv	&& echo $TEST_ENV
[14:44:07.200] #11 sha256:e039085acbb76b7459e24bf8509dd40c691b9f8c3130e169fc0f714cfd3fb2bb
[14:44:07.475] #11 0.408 NODE_VERSION=19.2.0
[14:44:07.643] #12 exporting to image
[14:44:07.643] #11 0.408 YARN_VERSION=1.22.19
[14:44:07.670] #12 sha256:e8c613e07b0b7ff33893b694f7759a10d42e180f2b4dc349fb57dc6b71dcab00
[14:44:07.674] #11 0.408 SHLVL=1
[14:44:07.674] #12 exporting layers 0.0s done
[14:44:07.683] #11 0.408 HOME=/home/node
[14:44:07.684] #12 writing image sha256:744dfe80c838455770c6171c995bd0fcea296617ea795669d756a821640611a7 done
[14:44:07.702] #11 0.408 PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
[14:44:07.705] #12 naming to docker.io/library/clbauc5ht001jmb930088h78i:4b8d074 done
[14:44:07.706] #11 0.408 PWD=/home/node
```

### Version

3.11.13

**Comment by andrasbacsai** [comment]
Can you please recheck again? I slightly modified the secrets part.

**Comment by Aviortheking** [comment]
Nope, still seems to be happening :(

**Comment by Aviortheking** [comment]
issue fixed as of `v3.12.2`
